config/nvim/init.vim

```diff
@@ -0,0 +1,3 @@
+    set runtimepath^=~/.vim runtimepath+=~/.vim/after
+    let &packpath = &runtimepath
```

Comment:
What do these two lines achieve?

Reply:
Sorry, replied from email...

No idea, it's just what neovim suggested.